src/weatherdata/ipm.py

```diff
@@ -11,30 +11,30 @@
 import pandas
 import datetime
 
-ipm = IPM()
-ressource_ids = {'Met Norway Locationforecast': 'yr',
-                 'Finnish Meteorological Institute measured data': 'fmi',
-                 'FMI weather forecasts': 'fmi_forecasts',
-                 'Landbruksmeteorologisk tjeneste': 'lmt',
-                 'MeteoBot API': 'metobot',
-                 'Fruitweb': 'fw',
-                 'Metos':'metos'}
-
-weather_adapter = {'fmi': ipm.get_weatheradapter_fmi,
-                   'yr': ipm.get_weatheradapter_yr,
-                   'lmt':'',
-                   'meteobot':'',
-                   'fw':'',
-                   'metos':''
-                    }
+#ipm = IPM()
```

Comment:
It is easier to comment with triple quotes : """
Thus you just have to add/rm two lines to manage the "dead" code